### PR TITLE
Mersenne Prime Number

### PR DESCRIPTION
**Pull request by KB786**
Code to print the Mersenne Prime numbers up to the range you set with just an input and a click

### CODE REVIEW

File: mersenneprime.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+//To find mersenne's prime upto 
+
```
Comment (sachin235) [inline mersenneprime.java:2]: Please briefly describe what is Mersenne's Prime number.

**Review by sachin235 — CHANGES_REQUESTED** [review]

File: mersenneprime.java (hunk 1 above)
Comment (sachin235) [reply mersenneprime.java:2]: Also, it would be great if you could add a sample input and output as comments to make it more clear.

**Comment by sachin235** [comment]
@KB786 move the file to the Mathematics folder in Java.

**Comment by KB786** [comment]
Is my PR merged ?

**Pull request merged**